examples/run_glue_trainer.py

```diff
@@ -0,0 +1,503 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Entrypoint that runs the Composer trainer on a provided YAML hparams file.
+
+Specifically designed for the NLP use case, allowing pre-training and fine-tuning on
+downstream tasks to be handled within one script. This script requires that the
+run_composer_trainer.py script lies in the same folder as this one.
+
+Example that pretrains a BERT::
+    >>> composer examples/run_glue_trainer.py
+    -f examples/glue/glue_example.yaml
+    --training_scheme pretrain
+
+Example that pretrains and finetunes a BERT::
+    >>> composer examples/run_glue_trainer.py
+    -f examples/glue/glue_example.yaml
+    --training_scheme all
+
+Example that finetunes a pretrained BERT::
+
+    >>> composer examples/run_glue_trainer.py
+    -f examples/glue/glue_example.yaml
+    --training_scheme finetune
+"""
+import multiprocessing as mp
+import os
+import subprocess
+import sys
+import tempfile
+import warnings
+from concurrent.futures import ProcessPoolExecutor as Pool
+from dataclasses import dataclass
+from multiprocessing.context import BaseContext
+from typing import Dict, List, Optional, Tuple
+
+import torch
+import yahp as hp
+import yaml
+from tabulate import tabulate
+
+import examples.glue.nlp_trainer_hparams.GLUETrainerHparams as GLUETrainerHparams
```

Comment:
Can you put this file in the `glue` folder, and try:
```suggestion
from .nlp_trainer_hparams import GLUETrainerHparams, NLPTrainerHparams
```